### DISCUSSION TITLE
ssh: gcr-ssh-agent socket not available inside the sandbox

### DISCUSSION THREAD

**Discussion by michelesr** [netblue30/firejail#6242] [Q&A] [answered]
Had to whitelist it in my local profile... since GNOME keyring seems to have dropped the ssh agent feature and moved it to gcr-ssh-agent, would it be appropriate to whitelist it in the default profile?

See https://wiki.archlinux.org/title/GNOME/Keyring%20#SSH_keys for more info.

```
whitelist /run/user/1000/gcr
```

**Comment by ghost** [comment]
Thanks for the info! I'm not sure if we should add `whitelist ${RUNUSER}/gcr` to `ssh.profile` though. As other apps use GNOME keyring it might make more sense to add it to `whitelist-runuser-common.inc` instead. This upstream change will affect `ssh-agent.profile` too I suppose.

If you could open a PR for this we'll see what other collaborators think about a good, stable fix for the socket.

**Comment by kmk3** [comment, marked as answer]
Added on:

* #6258
